projects/nmap.org/package.yml

```diff
@@ -0,0 +1,32 @@
+distributable:
+  url: https://nmap.org/dist/nmap-{{version.raw}}.tgz
+  strip-components: 1
+
+versions:
+  - 7.93
+
+dependencies:
+  openssl.org: '*'
```

Comment:
Yeah, probs not true though. At some point we will package openssl 3 and I'm pretty sure it’s API is very different to 1.1.1